lib/ansible/executor/process/worker.py

```diff
@@ -125,6 +113,51 @@ def _hard_exit(self, e):
 
         os._exit(1)
 
+    @contextlib.contextmanager
```

Comment:
This was originally a contextmanager, but doesn't need to be any longer, although it's still left as one for now.